<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1066</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_drib">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
textcolour:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>Кількість створених об’єктів 0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="text_dobutok">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="text_skr2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Bob1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Оберений дріб №1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="text_skr1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>==</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Bob2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Оберений дріб №2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="text_inv1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Bdobutok">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Множення</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="text_divide">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="text_add">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="Bskr2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Скорочений дріб №2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Badd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Додавання</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>64</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">	background-color:rgb(255, 255, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Дріб №2</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QPushButton" name="Bskr2_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Очистити всі поля</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="text_inv2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Bdivide">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ділення</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="drib2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>64</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Bminus">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Віднімання</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Bskr1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Скорочений дріб №1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>64</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">	background-color:rgb(255, 255, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Дріб №1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="drib1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>64</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="text_minus">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" rowspan="3">
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>140</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">	background-color:rgb(255, 255, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Дріб №2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" rowspan="3">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>140</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">	background-color:rgb(255, 255, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Дріб №1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" rowspan="3">
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>140</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>70</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>779</x>
      <y>700</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
